src/Microsoft.Health.Dicom.Tests.Common/Microsoft.Health.Dicom.Tests.Common.csproj

```diff
@@ -6,11 +6,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Ensure.That" Version="10.1.0" />
     <PackageReference Include="fo-dicom" Version="$(FoDicomVersion)" />
+    <PackageReference Include="fo-dicom.Json" Version="$(FoDicomVersion)" NoWarn="NU1701" />
+    <PackageReference Include="Microsoft.Extensions.Primitives" Version="$(SdkPackageVersion)" />
+    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="17.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="NSubstitute" Version="4.2.2" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
```

Comment:
When I remove this, I get an error in Visual Studio test discovery: 
```
Microsoft.VisualStudio.TestPlatform.ObjectModel.TestPlatformException: Unable to find C:\Users\jovinson\Source\Repos\dicom-server\src\Microsoft.Health.Dicom.Tests.Common\bin\Debug\net5.0\testhost.dll. Please publish your test project and retry.
   at Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Hosting.DotnetTestHostManager.GetTestHostPath(String runtimeConfigDevPath, String depsFilePath, String sourceDirectory)
   at Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Hosting.DotnetTestHostManager.GetTestHostProcessStartInfo(IEnumerable`1 sources, IDictionary`2 environmentVariables, TestRunnerConnectionInfo connectionInfo)
   at Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Client.ProxyOperationManager.SetupChannel(IEnumerable`1 sources, String runSettings)
   at Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Client.ProxyDiscoveryManager.DiscoverTests(DiscoveryCriteria discoveryCriteria, ITestDiscoveryEventsHandler2 eventHandler)
```

Things still run but it's messy.

Reply:
This test exploration is caused by the xunit import. The project should really import `xunit.assert` instead of the entire test sdk. Otherwise Visual Studio will think it's a unit test project

Reply:
New output for running the core tests:
> Building Test Projects
Starting test discovery for requested test run
========== Starting test discovery ==========
[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET 6.0.0-rtm.21522.10)
[xUnit.net 00:00:02.67]   Discovering: Microsoft.Health.Dicom.Core.UnitTests
[xUnit.net 00:00:03.08]   Discovered:  Microsoft.Health.Dicom.Core.UnitTests
========== Test discovery finished: 604 Tests found in 4.5 sec ==========
========== Starting test run ==========
[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET 6.0.0-rtm.21522.10)
[xUnit.net 00:00:00.64]   Starting:    Microsoft.Health.Dicom.Core.UnitTests
[xUnit.net 00:00:00.87]     Microsoft.Health.Dicom.Core.UnitTests.Features.Retrieve.RetrieveResourceHandlerTests.GivenIncorrectTransferSyntax_WhenRetrievingStudy_ThenDicomBadRequestExceptionIsThrownAsync [SKIP]
[xUnit.net 00:00:00.87]       Move this tests to move this tests to RetriveResourceService, since the logic to validate TransferSyntax has moved there
[xUnit.net 00:00:01.20]     Microsoft.Health.Dicom.Core.UnitTests.Features.Retrieve.RetrieveTransferSyntaxHandlerTests.GivenMultipleMatchedAcceptHeadersWithDifferentQuality_WhenGetTransferSyntax_ThenShouldReturnLargestQuality [SKIP]
[xUnit.net 00:00:01.20]       Will be enabled later as https://microsofthealth.visualstudio.com/Health/_workitems/edit/75782
[xUnit.net 00:00:09.82]   Finished:    Microsoft.Health.Dicom.Core.UnitTests
========== Test run finished: 1135 Tests (1133 Passed, 0 Failed, 2 Skipped) run in 9.8 sec ==========